nextflow.config

```diff
@@ -1,30 +1,49 @@
 params {
-    // required files
-    gut_mapping           = "$projectDir/data/known_and_found_gut_genomes.tsv"
-
-    // names for generated intermidiate files 
-    input_accessions      = 'input_accessions.tsv'
-    catalogue_metadata    = 'all_catalog_metadata.tsv'
+    // Accession mapping file for "human gut" catalogue of MGnify - DO NOT MODIFY OR DELETE
+    // This file complements metadata for the "human gut" catalogue of MGnify by converting
+    // custom MGYG accessions (which cannot be used with ENA API) to ENA-compatible accessions.
+    // Essential for proper metadata retrieval and processing of genomes from this catalogue.
+    gut_accessions_mapping = "${projectDir}/data/known_and_found_gut_genomes.tsv"
 
     // user input args
-    batch_size            = 250
-    previous_table        = null
-    processed_acc         = null
-    external_input        = null
-    external_metadata     = null
-    cleanup               = true
-    debug                 = false
-    output_path           = "$PWD/results"
+    batch_size             = 250
+    merge_with_results     = null
+    skip_accessions        = null
+    accessions_list        = null
+    catalogues_metadata    = null
+    cleanup                = true
+    debug                  = false
+    output_path            = "${System.getenv('PWD')}/results"
```

Comment:
I think you can use `${workDir}` instead of `${System.getenv('PWD')}` 

docs about this: https://www.nextflow.io/docs/latest/reference/stdlib-namespaces.html